src/transactions/test/InvokeHostFunctionTests.cpp

```diff
@@ -2514,9 +2544,9 @@ TEST_CASE("temp entry eviction", "[tx][soroban]")
         cfg.DEPRECATED_SQL_LEDGER_STATE = !enableBucketListDB;
         cfg.BACKGROUND_EVICTION_SCAN = backgroundEviction;
 
-        // overrideSorobanNetworkConfigForTest commits directly to the database,
-        // will not work if BucketListDB is enabled so we must use the cfg
-        // override
+        // overrideSorobanNetworkConfigForTest commits directly to the
```

Comment:
I think we can remove this now.

Reply:
#Hmm, I haven't changed anything around that logic, have I? Or should this be generally addressed in some other change set?